config/benchmarkssuperglue/gap_bert.conf

```diff
@@ -0,0 +1,39 @@
+// This imports the defaults, which can be overridden below.
+include "../defaults.conf"  // relative path to this file
+
+project_dir = ${JIANT_PROJECT_PREFIX}
+exp_name = "bert_gap_running"  // configure this
```

Comment:
What does 'running' mean?

Reply:
Sorry, that was bad naming. Changed the names to be more informative.